src/content/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-tvos-crash-reporting.mdx

```diff
@@ -31,6 +31,7 @@ For more information, see [Retrieve and download dSYMs](/docs/mobile-monitoring/
 
 Crash reporting is enabled by default, but there are some circumstances where it will be disabled:
 
+* **If running in simulator:** A build running through the simulator cannot be symbolicated.
```

Comment:
@cdillard-NewRelic Can we make this into a full phrase? i.e., "If the app is running in a simulator:"

Reply:
Yeah your edit plus one clarification in the last part looks good. 👍 
```suggestion
* **If the app is running in a simulator:** An app running through the simulator cannot be used with crash reporting. The simulator crashes cannot be symbolicated.
```